public/manifest.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Karantenehjelpen",
+  "short_name": "Karantenehjelpen",
+  "description": "Motta hjelp til å handle hvis du er i karantene. Hjelp de som sitter i karantene med å handle",
```

```suggestion
  "description": "Motta hjelp til å handle hvis du er i karantene, eller hjelp de som sitter i karantene med å handle.",
```

Litt usikker på hvordan det skal formuleres.

---

Virker som en god endring. Er såklart åpen for å endre til noe helt annet også hvis det ønskes :)

---

Endret den til skrivemåten din :smile: